props/prop_base.h

```diff
@@ -607,7 +622,11 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   void PollScanId() {
     if (find_blade_again_pending_) {
       find_blade_again_pending_ = false;
+      blade_id_should_report_= true;
       FindBladeAgain();
+      // Indicate what happened when different Blade ID detected
+      SaberBase::DoNewFont();
```

Comment:
Why is this needed?
Doesn't FindBladeAgain() already do this? (or something similar?)

Reply:
FindBladeAgain() was just silently running.
This was the only indication something occurred:
```
22:56:52.786 -> blade = 0
22:56:52.786 -> WS2811 Blade with 11 leds.
22:56:52.786 -> WS2811 Blade with 5 leds.
22:56:52.786 -> WS2811 Blade with 11 leds.
22:56:52.786 -> Simple Blade
22:56:52.786 -> Simple Blade
22:56:52.786 -> Style RAM = 72
22:56:52.786 -> Style RAM = 72
22:56:52.786 -> Style RAM = 72
22:56:52.786 -> Style RAM = 72
22:56:52.786 -> Style RAM = 72
22:56:52.786 -> Scanning sound font: testfont2b
22:56:52.888 ->  done
22:56:52.888 -> Scanning sound font: common
22:56:52.888 ->  done
22:56:52.888 -> Scanning sound font: commonBU
22:56:52.961 ->  done
22:56:53.140 -> Activating polyphonic font.
22:56:53.242 -> Activating SmoothSwing V2
22:56:53.242 -> Accent Swings Enabled.
22:56:53.242 -> Polyphonic swings: 1
22:56:53.242 -> Monophonic swings: 0
22:56:53.242 -> Accent Slashes Enabled.
22:56:53.242 -> Polyphonic slashes: 1
```
so it made sense to play the font.wav and show the preset name, no?
Strange though, I think FindBladeAgain USED to do that though?

Reply:
Update - I think just calling SetPreset(0, true) does this instead. 
Edited.